odoo/addons/base/models/ir_attachment.py

```diff
@@ -336,7 +336,10 @@ def _postprocess_contents(self, values):
                     nw, nh = map(int, max_resolution.split('x'))
                     if w > nw or h > nh:
                         img = img.resize(nw, nh)
-                        quality = int(ICP('base.image_autoresize_quality', 80))
+                        if _subtype == 'png':
```

Comment:
> supported_subtype = ICP('base.image_autoresize_extensions', 'png,jpeg,bmp,tiff').split(',')

I see that webp is not listed in there in master, is that another bug or am I missing something?